.github/actions/deploy-klt-on-cluster-scheduling-gates/action.yml

```diff
@@ -0,0 +1,76 @@
+name: "Deploy Keptn Lifecycle Toolkit on GH cluster"
```

Comment:
could we also just add a new input for enabling the scheduling gates to the existing installation action: https://github.com/keptn/lifecycle-toolkit/blob/main/.github/actions/deploy-klt-on-cluster/action.yml?